chromium_src/chrome/utility/services.cc

```diff
@@ -7,11 +7,28 @@
 
 #if !defined(OS_ANDROID)
 #include "brave/utility/importer/brave_profile_import_impl.h"
-#define ProfileImportImpl BraveProfileImportImpl
+
+namespace {
+
+auto RunBraveProfileImporter(
```

Comment:
we also need `!defined(OS_ANDROID)` guard for this function

Reply:
Done.